build.rs

```diff
@@ -78,8 +78,18 @@ fn main() {
 	let bindings_out_path = PathBuf::from(env::var("OUT_DIR").unwrap()).join("bindings.rs");
 	std::fs::write(bindings_out_path, rust_source).expect("Couldn't write updated bindgen output");
 
+	// The nrf9160 feature is on by default for legacy support, so we check for the nrf9120 feature
+	// first to handle the case in which both nrf9120 and nrf9160 are set (i.e., nrf9120 is set,
+	// but no-defaults is not set)
```

Comment:
This seems a bit weird... Having both 9160 and 9120 on at the same time should be a compile error too

Reply:
Agreed.  I went back and forth on this.  I was aiming to backward compatible.